vocabularies/UI.xml

```diff
@@ -993,6 +993,11 @@ The trend is
         <Annotation Term="Core.Description" String="Facets that reference UI.FieldGroup annotations to group filterable fields" />
       </Term>
 
+      <Term Name="OperationParameterFacets" Type="Collection(UI.ReferenceFacet)" Nullable="false" AppliesTo="Action Function ActionImport FunctionImport">
```

Comment:
Please add `ActionImport` to the `AppliesTo` list of `FieldGroup` here:

https://github.com/SAP/odata-vocabularies/blob/4b48175db142914aa723129920ca58bcebb08c0f/vocabularies/UI.xml#L157

Reply:
done